mobile/src/src/components/Housing/Listings/ListingView.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { StyleSheet, Dimensions, Image, View, Text } from "react-native";
+import { Card, Title, Paragraph, Avatar } from 'react-native-paper';
+import ProfilePhoto from "../../common/ProfilePhoto";
+import { Choices } from "../../../utils/endpoints";
+
+const ListingView = (props) => {
+
+    const {width, height} = Dimensions.get('window');
+
+    return (
+        <Card style={styles.container}>
+            <Card.Content>
+                <View style={{ flexDirection: "row" }}>
+                    <View style={{flex: 1}}>
+                        <ProfilePhoto photo={props.creator.photo}></ProfilePhoto>
+                    </View>
+                    <View style={{flex: 5, flexDirection: "column"}}>
```

Comment:
We can probably add this style to styles variable.

Reply:
Did this get fixed?

Reply:
disregard